dotcom-rendering/docs/principles/browser-support.md

```diff
@@ -32,8 +32,7 @@ looks like:
 | UC Browser        | 12          | 0.30%            |
 | Opera             | 56          | 0.25%            |
 
-The above table was calculated using [this spreadsheet](https://docs.google.com/spreadsheets/d/17g8py3P30nJLVqrlgc8JW4rkFhPTqNvwrMJdaOFnFzs/)
-based on data from Google Analytics.
+The above table was calculated based on data from Google Analytics.
```

Comment:
Why remove the link to the spreadsheet, here? It's valuable historical information, even if we no longer use GA.

Reply:
I wasn't planning to remove it because of the same reason you mentioned but the link is broken!

![image](https://github.com/guardian/dotcom-rendering/assets/23424805/b8792471-7e18-42c1-b8d2-112e25cc6a84)

Reply:
Aside: I'd favour linking to the central [`browserslist-config` docs](https://github.com/guardian/csnx/tree/main/libs/%40guardian/browserslist-config). I can think of no reason for there to be 2 separate lists of supported browsers.

Reply:
[Related](https://github.com/guardian/dotcom-rendering/blob/ce71b78603aa13ca45b9aa222d2fae66b4ba9d7f/dotcom-rendering/docs/principles/browser-support.md?plain=1#L5)

```
**Note**: This documentation is out of date and requires and update, please badger the team to do that if you're reading this. GT - 23/06/2020
```